docs/running-clusterfuzzlite/github_actions.md

```diff
@@ -6,7 +6,191 @@ title: GitHub Actions
 nav_order: 1
 permalink: /running-clusterfuzzlite/github-actions/
 ---
-
 # GitHub Actions
+{: .no_toc}
+
+- TOC
+{:toc}
+---
+
+
+ClusterFuzzLite can be configured in a number of ways to enable both PR and
+batch fuzzing.
+
+We recommend having two workflow files:
+
+- `.github/workflows/cflite_continuous.yaml` (for building and batch fuzzing)
+- `.github/workflows/cflite_pr.yaml` (for PR fuzzing)
+
+TODO: Host a clean, complete example somewhere.
+
+## Continuous builds (required)
+
+Continuous builds are used whenever a crash is found during PR or batch fuzzing
+to determine if this crash was newly introduced.
+
+Add the following to `cflite_continuous.yaml`:
+
+{% raw %}
+```yaml
+name: CIFuzz continuous
+on:
+  push:
+    branches:
+      - main
+jobs:
+  Build:
+   runs-on: ubuntu-latest
+   steps:
+   - name: Build Fuzzers
+     id: build
+     uses: google/clusterfuzzlite/actions/build_fuzzers@v1
+     with:
+       upload-build: true
```

Comment:
It works without it. I think because of what's discussed here:  https://twitter.com/BRIAN_____/status/1392234974783279105?s=19 

A separate undocumented token (which is included in your ported code) is used for uploading artifacts.

We can probably actually get rid of the github-token input variable as well, since the only upload code was relying on it.